extract/extract.py

```diff
@@ -334,7 +349,7 @@ def extract_office(self, request: ServiceRequest, local, encoding: str):
 
         out_name, password = res
         self._last_password = password
-        display_name = "_decoded".join(os.path.splitext(os.path.basename(request.file_path)))
+        display_name = request.file_name
```

Comment:
looks like you gotta rebase this branch on the updated master, as this commit is already in master